phpunit.xml.dist

```diff
@@ -12,4 +12,7 @@
             </exclude>
         </whitelist>
     </filter>
+    <php>
+        <const name="UNIT_TESTS_EXECUTION" value="true"/>
```

Comment:
I'd prefer not to rely on variables set outside of PHP code for this - but we can discuss this if there's a valid reason to keep the code that uses it (see https://github.com/silverstripe/silverstripe-queuedjobs/pull/467#discussion_r2501300088)